src/components/ModelViewer/Error404.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import imgerror  from '../ModelViewer/error404img.jpg';
+import '../ModelViewer/Error404_style.css';
+
+const Error404 = () => {
+
+    const redirectToHomePage = () => {
+        window.location.href = '/'; // Replace with the desired home page URL or path
+      };
+
```

Comment:
Please change the above function and redirect using  useNavigate ( from react-router-dom )
Ref : https://stackoverflow.com/questions/34735580/how-to-do-a-redirect-to-another-route-with-react-router